internal/cli/atlas/deployments/connect.go

```diff
@@ -48,7 +48,7 @@ func ConnectBuilder() *cobra.Command {
 			return opts.PreRunE(
 				opts.InitOutput(cmd.OutOrStdout(), ""),
 				opts.InitInput(cmd.InOrStdin()),
-				opts.InitStore(podman.NewClient(log.IsDebugLevel(), log.Writer())),
+				opts.InitStore(cmd.Context(), podman.NewClient(log.IsDebugLevel(), cmd.OutOrStdout())),
```

Comment:
```suggestion
				opts.InitStore(cmd.Context(), podman.NewClient(log.IsDebugLevel(), cmd.ErrOrStderr())),
```
or 
```suggestion
				opts.InitStore(cmd.Context(), podman.NewClient(log.IsDebugLevel(), log.Writer())),
```
we want debug logs to go to Stderr